### ISSUE TITLE
S3BotoStorage import error on rendervarations

### ISSUE THREAD

**Issue by dporru** [codingjoe/django-stdimage#70]
Running the `rendervariations` command with `S3BotoStorage` as storage backend gives the following error:

```
ImportError: Module "django.core.files.storage" does not define a "S3BotoStorage" attribute/class
```

I'm on `django 1.9.2` with `django-storages-redux 1.4`, `python 1.3.5` and the current master of `django-stdimage`.

On inspection it seems the `field.storage` object in the `render` function in `stdimage/management/commands/rendervariations.py` prints as `<storages.backends.s3boto.S3BotoStorage object at 0x102f69550>`, but the `__module__` is `django.core.files.storage`.

The `class_to_path` function returns `'django.core.files.storage.S3BotoStorage'` when the actual class is located at `'storages.backends.s3boto.S3BotoStorage'`.

Any idea what is going wrong here?

**Comment by codingjoe** [comment]
Hi @dporru,
I must admit I haven't this one before, but I don't remember actually executing `rendervariations` on a production Django 1.9 project yet.

This is really odd. Can you post your storage config (without private keys please ;).

Without any deeper inspection, I would point the finger on [the lazy loading of the default storage](https://github.com/django/django/blob/master/django/core/files/storage.py#L344-L348).

**Comment by dporru** [comment]
Thanks for the quick response! My storage config:
```
DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
STATICFILES_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
```
I think your correct about the lazy loading of `DefaultStorage` as a cause. `field.storage._wrapped` gives back the correct `storages.backends.s3boto.S3BotoStorage` object.

A naive fix would be to check for the `cls._wrapped` property in `class_to_path` and use that in stead of `cls` when it is present.

**Comment by codingjoe** [comment]
It's still a curious error. Since the command import the storage anywhere but just forwards it.
Do you have a full stack trace? Where is the error being raised?

**Comment by dporru** [comment]
Here's the full stack trace. It doesn't contain any secrets :wink:
https://gist.github.com/dporru/915d8a448d90b4b08e5b

**Comment by codingjoe** [comment]
Ok, it seems it is actually [this bit](https://github.com/codingjoe/django-stdimage/blob/master/stdimage/management/commands/rendervariations.py#L67) that breaks.

**Comment by dporru** [comment]
Yes exactly.
-- 
Verzonden van mijn Android telefoon met K-9 Mail.

**Comment by codingjoe** [comment]
Fix released in 2.2.1

**Comment by dporru** [comment]
Thanks!
-- 
Verzonden van mijn Android telefoon met K-9 Mail.
